src/components/Code.tsx

```diff
@@ -204,7 +202,7 @@ function CodeRef({
 ref: RefObject<any>) {
   const parentFillLevel = useFillLevel()
   const tabStateRef = useRef()
-  const [selectedTabKey, setSelectedTabKey] = useState<Key>(makeTabKey((tabs && tabs[0]) || {}))
+  const [selectedTabKey, setSelectedTabKey] = useState<Key>((tabs && tabs[0]?.key) || '')
```

Comment:
could this actually be `tabs?[0]?.key || ''` or is that too wild

Reply:
I keep wanting to do that but typescript (or the linter, or something) yells at me.